config/crd/v1/bases/velero.io_podvolumerestores.yaml

```diff
@@ -16,7 +16,37 @@ spec:
     singular: podvolumerestore
   scope: Namespaced
   versions:
-  - name: v1
+  - additionalPrinterColumns:
+    - description: Namespace of the pod containing the volume to be restored
+      jsonPath: .spec.pod.namespace
+      name: Namespace
+      type: string
+    - description: Name of the pod containing the volume to be restored
```

Comment:
Do we need this?
if I use `kubectl` will it print the `Name` column twice?

Reply:
It's the name of the pod not the name of PVR, and it shows as column `Pod` when running `kubectl`